docs/topics/logging.txt

```diff
@@ -261,7 +267,29 @@ First, here's a simple configuration which writes all request logging from the
 If you use this example, be sure to change the ``'filename'`` path to a
 location that's writable by the user that's running the Django application.
 
-Second, here's an example of a fairly complex logging setup, configured using
+Second, here's an example of how to make the logging system print everything to
+the console based on the value of an environment variable . It may be useful
+during local development. Set the environment variable
+``DJANGO_LOG_LEVEL=DEBUG`` to see all of Django's logging calls in the console::
+
+    # Not working yet. Entries from django.request and django.security aren't
+    # displayed.
+    LOGGING = {
+        'version': 1,
+        'handlers': {
+            'console': {
+                'level': os.getenv('DJANGO_LOG_LEVEL', 'INFO'),
+                'class': 'logging.StreamHandler',
+            },
+        },
+        # A catch-all root logger.
+        'root': {
+            'handlers': ['console'],
+            'level': 'DEBUG',
+        },
+    }
+
+Finally, here's an example of a fairly complex logging setup, configured using
 :func:`logging.config.dictConfig`::
 
     LOGGING = {
```

Comment:
Hmm, this example also uses `disable_existing_loggers: True`, which is bad. I don't think we should show any examples in our docs with `disable_existing_loggers`. This example should perhaps also be changed to set `LOGGING_CONFIG` to `None` and use `dictConfig` directly in order to get the desired effect? (Although in this case since the default logger for `django.request` is redefined here, maybe that re-enables it anyway?)